deployment/network-operator/values.yaml

```diff
@@ -164,6 +167,14 @@ sriov-network-operator:
       beta.kubernetes.io/os: "linux"
       network.nvidia.com/operator.mofed.wait: "false"
 
+nic-configuration-operator-chart:
+  operator:
+    image:
+      tag: v0.1.0
```

Comment:
Please add the versions to the `hack/release.yaml`

Reply:
done